applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp

```diff
@@ -48,8 +48,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
                    std::unique_ptr<IntegrationCoefficientModifier> pCoefficientModifier = nullptr)
         : Element(NewId, ThisNodes),
           mpStressStatePolicy{std::move(pStressStatePolicy)},
-          mpIntegrationCoefficientsCalculator{
-              std::make_unique<IntegrationCoefficientsCalculator>(std::move(pCoefficientModifier))}
+          mIntegrationCoefficientsCalculator{IntegrationCoefficientsCalculator(std::move(pCoefficientModifier))}
```

Comment:
We can simply pass the constructor's argument (there is no need to construct a temporary calculator instance):
```suggestion
          mIntegrationCoefficientsCalculator{std::move(pCoefficientModifier)}
```

This suggestion also applies to lines 62 and 74.

Reply:
Thank you for pointing. Indeed the calculator exists already.